cmd/mount_unix.go

```diff
@@ -141,6 +141,10 @@ func fuseFlags() []cli.Flag {
 			Name:  "root-squash",
 			Usage: "mapping local root user (uid = 0) to another one specified as <uid>:<gid>",
 		},
+		&cli.StringFlag{
```

Comment:
We should make flag `non-default-permission` visible or enable it when squash all users. Otherwise, the kernel may deny users to modify files they created.

```
$ touch hello
touch: setting times of 'hello': Permission denied
```

Reply:
We probably have to always disable default permission to make the `all-squash` work

Reply:
No, there will be lots of access() and much slower.

Reply:
But `all-squash` can hardly work with `default-permission`.

Reply:
When defaultPermission is disabled, we should make sure that the permission is checked in all the places, or access is called for them.